src/cache.js

```diff
@@ -334,10 +341,16 @@ class Wrapper {
   }
 
   async get (key) {
+    if (this.transformer) {
+      return this.transformer.deserialize(this.storage.get(key))
```

Comment:
```suggestion
      return this.transformer.deserialize(await this.storage.get(key))
```

Reply:
Good catch, thanks! I changed this, but sadly the error remains. I've also added documentation and an example to the README.

Reply:
Okay, I've managed to fix all the errors. There actually was another bug in the implementation.